data/monster/Arena/greenhorn/achad.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<monster name="Achad" nameDescription="achad" race="blood" experience="70" speed="185" manacost="0">
+<monster name="Achad" nameDescription="Achad" race="blood" experience="70" speed="185" manacost="0">
```

Comment:
Strings inside ``nameDescription=""`` are no longer case-sensitive -- all strings are converted to lowercase.